src/modules/News/index.tsx

```diff
@@ -61,61 +64,69 @@ export function NewsUI(props: NewsUIProps): ReactElement {
     } = props;
     const wrapMsg = (msg: string) => <div className="flex flex-1 items-center justify-center text-gray-600">{msg}</div>;
     return (
-        <div className="h-full flex flex-col news">
-            <div className="flex flex-col pl-3 pr-3 pt-3 shadow-3xl news__header">
-                <div className="flex items-center mb-3">
-                    <Button className="mr-2" onClick={onBack}>
-                        <ArrowLeft className="fill-current text-black w-3.5 z-1 relative mr-2 group-active:fill-current group-active:text-white" />
-                        News
-                    </Button>
-                    <Input
-                        icon={<MagnifyingGlass />}
-                        name="search"
-                        placeholder="Search Article..."
-                        value={searchTerm}
-                        onChange={onChangeSearch}
-                    />
-                </div>
-            </div>
-            {searchTerm && (
-                <div className="bg-gray-50 flex h-10 items-center p-3 shadow sticky text-gray-500 text-sm top-3 z-10">
-                    <div className="text-sm">
-                        Showing {matches.length} result{matches.length === 1 ? '' : 's'} for &quot;
-                        <span className="font-semibold">{searchTerm}</span>
-                        &quot;
-                    </div>
-                    <div className="flex-1" />
-                    <button
-                        tabIndex={0}
-                        className="w-2.5 mr-2 cursor-pointer rotate-180 hover:text-gray-600"
-                        onClick={prevMatch}
-                    >
-                        <Chevron />
-                    </button>
-                    <div className="min-w-[35px] mr-2 text-center">
-                        {matchIndex + 1} / {matches.length}
+        <div className={classNames('h-full flex flex-col news', { dark: darkMode })}>
+            <div className="dark:bg-bluegray-7">
+                <div className="flex flex-col pt-3 px-3 shadow-3xl dark:shadow-3xl-dark dark:bg-bluegray-6 news__header">
+                    <div className="flex group items-center mb-3">
+                        <Button onClick={onBack}>
+                            <ArrowLeft className="fill-current w-3.5 z-1 relative mr-2 group-active:fill-current group-active:text-white" />
+                            News
+                        </Button>
+                        <Input
+                            className="mx-2"
+                            icon={<MagnifyingGlass />}
+                            name="search"
+                            placeholder="Search Article..."
+                            value={searchTerm}
+                            onChange={onChangeSearch}
+                        />
+                        <SettingsButton showTextSentiment={false} />
```

Comment:
This should probably have `showTonalSentiment={false}` as well right? I think we want that hidden even when we enable the feature later.

Reply:
I made `showTonalSentiment` false by default (instead of the `false &&...` in `SettingsButton` now) but can def make it true by default and pass false in all the modules that use it.

Reply:
Yea I think true by default and pass false in makes more sense, thanks!